pkg/console/operator/sync_v400.go

```diff
@@ -119,28 +118,20 @@ func (co *consoleOperator) sync_v400(ctx context.Context, controllerContext fact
 		return statusHandler.FlushAndReturn(oauthServingCertErr)
 	}
 
-	sec, secChanged, secErr := co.SyncSecret(ctx, set.Operator, controllerContext.Recorder())
-	toUpdate = toUpdate || secChanged
-	statusHandler.AddConditions(status.HandleProgressingOrDegraded("OAuthClientSecretSync", "FailedApply", secErr))
+	clientSecret, secErr := co.secretsLister.Secrets(api.TargetNamespace).Get(secretsub.Stub().Name)
+	statusHandler.AddConditions(status.HandleProgressingOrDegraded("OAuthClientSecretSync", "FailedGet", secErr))
 	if secErr != nil {
 		return statusHandler.FlushAndReturn(secErr)
 	}
 
-	oauthClient, oauthChanged, oauthErrReason, oauthErr := co.SyncOAuthClient(ctx, set.Operator, sec, consoleURL.String())
-	toUpdate = toUpdate || oauthChanged
-	statusHandler.AddConditions(status.HandleProgressingOrDegraded("OAuthClientSync", oauthErrReason, oauthErr))
-	if oauthErr != nil {
-		return statusHandler.FlushAndReturn(oauthErr)
-	}
-
 	actualDeployment, depChanged, depErrReason, depErr := co.SyncDeployment(
 		ctx,
 		set.Operator,
```

Comment:
not new, `set.Operator` is already deep copied because of the [this change](https://github.com/openshift/console-operator/pull/814/files#diff-d4d44dccf244addcf7cfdaf11b2e27171132f629a5c14a6382bdb712a82bedf5R307-R312), but we copy it again in `syncDeployment`. As things are, this resource gets deep copied 3 times:

- https://github.com/openshift/console-operator/pull/814/files#diff-d4d44dccf244addcf7cfdaf11b2e27171132f629a5c14a6382bdb712a82bedf5R308
- https://github.com/openshift/console-operator/pull/814/files#diff-d4d44dccf244addcf7cfdaf11b2e27171132f629a5c14a6382bdb712a82bedf5R323
- https://github.com/openshift/console-operator/pull/814/files#diff-b2ca349b96cbce4c3caf1c27d93f89b881f4e11ab0091498ec48e1bedcc25be2R255

Also, if all resources in the `set` arg to `synv_v400` are already deep copied, I think we can get rid of the `updatedOperatorConfig` arg, right?